test/integration/transactions/transactions.spec.test.js

```diff
@@ -90,7 +90,16 @@ const SKIP_TESTS = [
 
   // TODO(NODE-2034): Will be implemented as part of NODE-2034
   'Client side error in command starting transaction',
-  'Client side error when transaction is in progress'
+  'Client side error when transaction is in progress',
+
+  // TODO(NODE-5855): Unskip Transactions Spec Unified Tests mongos-unpin.unpin
+  'unpin after TransientTransactionError error on commit',
```

Comment:
I thought only this test was failing, did we observe the rest in this list also failing?

Reply:
At least two other transaction unpin tests are also failing: ['TransientTransaction error on abort'](https://spruce.mongodb.com/task/mongo_node_driver_next_rhel80_large_gallium_test_latest_sharded_cluster_aed1cf0d2b1083e24997e49bfe7f5416e944466e_24_01_26_16_43_30?execution=0&sortBy=STATUS&sortDir=ASC) , ['non-transient error on abort'](https://spruce.mongodb.com/task/mongo_node_driver_next_windows_vsCurrent_large_hydrogen_test_latest_sharded_cluster_aed1cf0d2b1083e24997e49bfe7f5416e944466e_24_01_26_16_43_30?execution=0&sortBy=STATUS&sortDir=ASC). I can skip those two in a follow-up ticket, or expand the scope of this ticket (edit the ticket description and AC).

Reply:
I am happy to skip what we need, but yea update the ticket so we can track, TIA